### DISCUSSION TITLE
Failed to save email settings

### DISCUSSION THREAD

**Discussion by ducnhoxlatui** [Budibase/budibase#10646] [Help] [answered]
Hi guys, 
I have the problem when saving email setting. I really need this to send email.

![image](https://github.com/Budibase/budibase/assets/71976420/62720ab6-2be4-4c75-8fd3-991168174636)

Thanks for helping me.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @ducnhoxlatui,

This seems to be an issue with your credentials to your gsmtp that you are using/setup. I would say you need to check that everything is setup correctly, I don't believe this to be an issue with Budibase. 

Trying giving this a read it may help. https://support.google.com/mail/thread/5621336?hl=en&msgid=6292199

I also ran through this entire process end to end using this tutorial on Youtube: https://www.youtube.com/watch?v=1YXVdyVuFGA&ab_channel=Sombex 

The setting used within Budibase are below.
<img width="647" alt="Screenshot 2023-05-22 at 08 14 10" src="https://github.com/Budibase/budibase/assets/126772285/af9b96bc-6693-4a83-9822-1006e153aa23">

The end result with it working below:
<img width="636" alt="Screenshot 2023-05-22 at 08 15 56" src="https://github.com/Budibase/budibase/assets/126772285/e5a6928b-3ef8-4c45-9595-37d6c7c7bb46">

Hopefully this helps you with getting this sorted!

**Reply by ducnhoxlatui** [reply]
oh, it works, i need to host smtp and create account for that
![image](https://github.com/Budibase/budibase/assets/71976420/9ce821bd-126c-4e13-a37a-009d9dd2edcc)

**Reply by ConorWebb96** [reply]
Yup this is exactly right! Glad you got it working!
